scripts/flag_validator.bash

```diff
@@ -5,7 +5,7 @@ SWUPD_DIR="$SCRIPTS_DIR"/..
 SWUPD="$SWUPD_DIR"/swupd
 
 swupd_commands=(info autoupdate check-update update bundle-add bundle-remove bundle-list bundle-info search-file diagnose repair os-install mirror clean hashdump 3rd-party)
```

Comment:
Suggestion: Maybe we have more comments for this file for certain parts to be easy to understand?

Reply:
The script does have some comments, I'll review it and see what other comments can be added to it to make it more straightforward, I'll do that in another PR if that is ok though.

Reply:
The script does have some comments, I'll review it and see what other comments can be added to it to make it more straightforward, I'll do that in another PR if that is ok though.